src/Definition/Source/SourceCache.php

```diff
@@ -23,21 +23,39 @@ class SourceCache implements DefinitionSource, MutableDefinitionSource
      */
     private $cachedSource;
 
+	/**
+	 * @var null|string
+	 */
+	private static $cacheKeyPrefix;
+
     public function __construct(DefinitionSource $cachedSource)
     {
         $this->cachedSource = $cachedSource;
     }
 
+	/**
+	 * @return string
+	 */
+    public static function getCacheKey(): string
```

Comment:
if `DefinitionSource` or `MutableDefinitionSource` has no `getCacheKey()` definition I would make it private.

Besides that adding a test case might be useful in the future

Reply:
it's used outside of the class

Reply:
```
		if(SourceCache::isSupported())
		{
			$output->write('Clearing apcu...');
			apcu_delete(new APCUIterator('~^' . SourceCache::getCacheKey() . '~'));
			$output->writeln('done.');
		}
```